internal/leakcheck/leakcheck.go

```diff
@@ -16,18 +16,81 @@
  *
  */
 
-// Package leakcheck contains functions to check leaked goroutines.
+// Package leakcheck contains functions to check leaked goroutines and buffers.
 //
 // Call "defer leakcheck.Check(t)" at the beginning of tests.
```

Comment:
There is no `Check` method anymore?

Reply:
Yeah you are right. I have updated this and provided a better method